Formula/c/cocoapods.rb

```diff
@@ -20,7 +20,7 @@ class Cocoapods < Formula
   uses_from_macos "libffi", since: :catalina
 
   def install
-    if MacOS.version >= :mojave && MacOS::CLT.installed?
+    if OS.mac? && MacOS.version >= :mojave && MacOS::CLT.installed?
       ENV["SDKROOT"] = ENV["HOMEBREW_SDKROOT"] = MacOS::CLT.sdk_path(MacOS.version)
     end
```

Comment:
We don't even use system Ruby here so this doesn't seem relevant anymore (+ is likely fixed in brew anyway).